db/database_test.go

```diff
@@ -674,28 +674,28 @@ func TestAllDocsOnly(t *testing.T) {
 	assert.Equal(t, 100, len(changes))
 
 	for i, change := range changes {
-		seq := i + 1
+		seq := i
 		if i >= 23 {
 			seq++
 		}
-		assert.Equal(t, SequenceID{Seq: uint64(seq)}, change.Seq)
+		assert.Equal(t, i != 99, fmt.Sprintf("alldoc-%02d", seq) == change.ID)
 		assert.Equal(t, i == 99, change.Deleted)
 		var removed base.Set
 		if i == 99 {
 			removed = channels.SetOf(t, "all")
+			assert.Equal(t, "alldoc-23", change.ID)
 		}
-		goassert.DeepEquals(t, change.Removed, removed)
+		assert.Equal(t, removed, change.Removed)
```

Comment:
As discussed, it would also be useful to verify during this iteration that sequences are ascending for all entries in the changes response.

Reply:
Added changes to check whether sequences are ascending for all entries in the changes response.